src/language/walker/ruleWalker.ts

```diff
@@ -82,6 +82,14 @@ export class RuleWalker extends SyntaxWalker {
         }
     }
 
+    public createFix(replacements: Replacement[]): Fix {
```

Comment:
I went back and forth on if we should have these wrapper methods at all (as the constructors have the exact same functionality). Do you think it makes sense to have them?